source/content/guides/decoupled-sites/site.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Pantheon Decoupled
+subtitle: 
+description: 
+categories: [platform]
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan]
+reviewed: "2022-07-31"
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled-sites/site
+anchorid: site
+---
+
+# Before You Begin
+
+You should have access to the Early Access Customer checklist provided during onboarding. To begin using Pantheon Decoupled, ensure you meet the following prerequisites:
+
+* Your repository is in GitHub.
+
+* You have a domain to use for the decoupled site.
+
+* You will use Next.js or Gatsby as the Frontend framework.
+
+* (Optional) You are using Drupal 9 or WordPress for the CMS.
+
+* (Optional) If using a preconfigured starter kit template, you will use one of the following combinations:
+
+    * Drupal 9 and Next.js
+
+    * WordPress and Gatsby
+
+    Note: Other configurations are not currently supported. 
```

Comment:
Should we add that they can experiment with unsupported front-end technologies with the import repo flow?